test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -365,13 +365,13 @@ def match_msg(self, lines, msg, allmatch=False, regexp=False,
         if lines:
             for l in lines:
                 if starttime is not None:
-                    # l[:19] captures the log record time
-                    tm = self.convert_date_time(l[:19])
+                    # l.split(';', 1)[0] gets the time stamp string
+                    tm = self.convert_date_time(l.split(';', 1)[0])
```

Comment:
Do the split only once so you can reuse the value in "if endtime is not None" block